TSC-Charter.md

```diff
@@ -36,12 +36,15 @@ TSC by voluntary resignation or by a standard TSC motion. A standard TSC motion
 can be used to change a regular TSC member to a voting TSC member, or to change
 a voting TSC member to a regular TSC member.
 
-No more than one-fourth of the TSC voting members may be affiliated with the
-same employer. If change in TSC voting membership or a change of
-employment by a TSC voting member creates a situation where more than
-one-fourth of the TSC voting membership shares an employer, then the situation
-must be immediately remedied by the removal of voting member status from one or
-more TSC voting members affiliated with the over-represented employer(s).
+In order to ensure that hostile takeovers of the Node.js project are not possible,
```

Comment:
I don't think we want to use the terminology _hostile takeover_ for several reasons. I think framing this as avoiding too much emphasis on the interests of a single stakeholder, or something like that. (Preferably something less wordy, but I'll take that if it's all we have.) _Hostile takeover_ has a specific technical/business meaning in English: ["the purchase of one business by another against the wishes of management"](https://en.wikipedia.org/wiki/Takeover#Hostile_takeover). That's not what we're talking about here (although I suppose you could say we're talking about something analogous) and I think it's also a bit...um...dramatic? It calls a lot of attention to this text in a way that we probably don't want to do.

That said, this is a non-blocking comment as the CPC will have to review this text too and I don't want to hold things up, and can leave my comment on the CPC issue/pull request.

Reply:
What term would you use?

Reply:
Maybe this?

```suggestion
In order to ensure adequate representation of the interests of diverse stakeholders,
```